.github/workflows/build-docker.yml

```diff
@@ -20,11 +21,31 @@ jobs:
           registry: ghcr.io
           username: ${{ github.repository_owner }}
           password: ${{ secrets.GITHUB_TOKEN }}
-      - name: Build and push
-        uses: docker/build-push-action@v2
+      - uses: docker/build-push-action@v2
         with:
           context: ci/tizen/docker
           file: ci/tizen/docker/Dockerfile
           push: true
-          tags: |
-            ghcr.io/${{ github.repository_owner }}/build-engine:latest
+          tags: ghcr.io/${{ github.repository_owner }}/build-engine:latest
+
+  testbed:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: build and push
+        env:
+          REPO_URL: http://download.tizen.org/releases/milestone/tizen/unified
+          BUILD_ID: tizen-unified_20211014.1
```

Comment:
How about using `latest`?

Reply:
I couldn't use `latest` because the `BUILD_ID` is also used in the file name:
```
${REPO_URL}/${BUILD_ID}/images/standard/${IMAGE}/${BUILD_ID}_${IMAGE}.tar.gz
```

Reply:
OK, thank you for answer.